ui/src/app/icons/edit/component.tsx

```diff
@@ -0,0 +1,77 @@
+import {Component} from '@angular/core';
+
+import * as React from 'react';
+
+import {ReactWrapperBase} from 'app/utils';
+
+interface EditComponentState { style: object; }
+
+const style = {
+  height: 14,
+  width: 14,
+};
+
+const fillStyle = {
+  fill: '#4D4CA5'
+};
+
+class EditComponentReact extends React.Component<{}, EditComponentState> {
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      style: style
+    };
+  }
+
+  mouseOver(): void {
+    this.setState({style: {...style, ...fillStyle}});
+  }
+
+  mouseLeave(): void {
+    this.setState({style: style});
+  }
+
+  render() {
+    return (
+      <svg style={this.state.style}
+           viewBox='0 0 14 14'
+           version='1.1'
+           xmlns='http://www.w3.org/2000/svg'
+           xmlnsXlink='http://www.w3.org/1999/xlink'
+           onMouseOver={() => this.mouseOver()}
+           onMouseLeave={() => this.mouseLeave()}>
+        <title>Edit</title>
+        <desc>Created with Sketch.</desc>
+        <defs></defs>
```

Comment:
It might be nice to run this through SVGO to strip out some of the cruft.

Reply:
This is a straight copy (modulo the react/lint changes) from the current `svg` - not sure how I feel about making that kind of change in a while-you're-at-it phase. We do have several of these, so maybe it makes sense to make that specific task.